apps/tasks/views.py

```diff
@@ -100,3 +113,74 @@ def perform_create(self, serializer):
         task_id = self.request.data['task']
         task = get_object_or_404(Task, id=task_id)
         send_notification([task.owner.email], "New comment!", "You task was commented")
+
+
+class TimerViewSet(viewsets.ModelViewSet):
+    permission_classes = [IsAuthenticated]
+    queryset = TimeLog.objects.all()
+    serializer_class = CreateTimeLogSerializer
+
+    def get_serializer_class(self):
+        if self.action == "perform_create":
+            return CreateTimeLogSerializer
+        if self.action == "add_time_log_manually":
+            return TimeLogSerializer
+        if self.action == "time_logs_by_id":
+            return TimeLogSerializer
```

Comment:
folosesti acelasi serializer se poate de scris elif self.action in ["add_time_log_manually", "time_logs_by_id"] return TimeLogSerializer